Automattic/wp-calypso
# Reorganise exports in lib/cart-values.

Exports were being grouped together into large default exports, as well as being exported individually. This change removes the large default exports, instead relying on the individual ones everywhere.

This started as an attempt to improve code-splitting, but that didn't pan out, unfortunately. The reorganised code is cleaner, though!

#### Changes proposed in this Pull Request

* Reorganize exports in `lib/cart-values`
* Reorganize exports in `lib/cart-values/cart-items`
* Update all dependant files accordingly

#### Testing instructions

* Test as much checkout/purchases functionality as you see fit. Breakage is unlikely, since mistakes would have most likely generated build-time errors. E2E tests should help, so I added the full suite.
